.gitignore

```diff
@@ -24,3 +24,4 @@
 *.ji
 
 .DS_Store
+get_toolchain.log
```

Comment:
So so complicated. I was about to give up programming and go back to a slide rule...